src/GoogleAdWordsEventForwarder.js

```diff
@@ -300,12 +334,19 @@
                 gTagScript.async = true;
                 gTagScript.onload = function () {
                     gtag('js', new Date());
-                    gtag('config', gtagSiteId);
+                    if (forwarderSettings.enableEnhancedConversions === 'True') {
+                        gtag('config', gtagSiteId, {
+                            allow_enhanced_conversions: true
+                        });
+                    } else {
+                        gtag('config', gtagSiteId);
+                    }
                     isInitialized = true;
                     processQueue(eventQueue);
                 };
                 gTagScript.src = 'https://www.googletagmanager.com/gtag/js?id=' + gtagSiteId;
                 document.getElementsByTagName('head')[0].appendChild(gTagScript);
+                gtag()
```

Comment:
Is it necessary to call gtag manually at the end like this? I thought the `onLoad` would handle that.

Reply:
Looks like this was never removed...good catch!